controllers/chaosimpl/networkchaos/partition/impl.go

```diff
@@ -360,12 +360,18 @@ func (impl *Impl) SetDrop(ctx context.Context, m *podnetworkchaosmanager.PodNetw
 		}
 		targetPods = append(targetPods, pod)
 	}
-	dstIpset := ipset.BuildIPSet(targetPods, externalCidrs, networkchaos, ipSetPostFix, m.Source)
-	m.T.Append(dstIpset)
+	dstSetIPSet, dstOtherIPSets := ipset.BuildIPSets(targetPods, externalCidrs, networkchaos, ipSetPostFix, m.Source)
+
+	for _, ipSet := range dstOtherIPSets {
+		m.T.Append(ipSet)
+	}
+
+	m.T.Append(dstSetIPSet)
+
 	m.T.Append(v1alpha1.RawIptables{
 		Name:      iptable.GenerateName(pbChainDirection, networkchaos),
 		Direction: chainDirection,
-		IPSets:    []string{dstIpset.Name},
+		IPSets:    []string{dstSetIPSet.Name},
```

Comment:
We should append `dstOtherIPSets` to here.  @YangKeao PTAL

Reply:
There is no need to - that's the whole point of `list:set` IP set. Matching against set type matches all other IP sets, so there is no need to duplicate iptables rules.

Reply:
OK, Got it, This is my fault! Can you add some comments to explain the different IPSet types?  I just found this in the test case.

Reply:
```
					{
						Name:     "ipset-set-name",
						Type:     "list:set",
						SetNames: []string{"set-1", "set-2"},
					},
					{
						Name:  "ipset-net-name",
						Type:  "hash:net",
						Cidrs: []string{"0.0.0.0/24"},
					},
					{
						Name: "ipset-net-port-name",
						Type: "hash:net,port",
						CidrAndPorts: []*pb.CidrAndPort{{
							Cidr: "1.1.1.1/32",
							Port: 80,
						}},
					},
``` 
From this case, the `list:set` is not an ipset type. It's just a set of ipset's name.

Reply:
I search the ipset doc, `list:set` also can be used as an ipset type. Ignore this `It is not appropriate to define it as an ipset type. ` advice.